src/openfl/_internal/renderer/opengl/GLRenderer.hx

```diff
@@ -124,9 +124,11 @@ class GLRenderer extends AbstractRenderer {
 		
 	}
 	
-
+	
+	var getMatrixHelperMatrix = new Matrix();
```

Comment:
I remember in `SubBitmapData` we defined/calld these helper variables like this:
```
static var __getPixelAbsoluteCoordsX:Int;
static var __getPixelAbsoluteCoordsY:Int;
```

Would it be a good idea to do the same here? Using a static and finding a consistent naming convention (I know this is not the strength of OpenFL...) for our helper variables.

Reply:
I mean it is used a bit different there and it's never given to the outside. Not sure, giving a static to the outside might not be the best. Naming migth still make sense

Reply:
Right, `static` is probably a good idea. I kind of rejected it when I was working at this initially because I confused this object with a `Matrix4` that we return and use later, so I wanted to be on the safe side (not really safe anyway), but since it's the object that we just use temporarily within this function it should be good.

Regarding the double-underscore prefix, I don't know. In general I don't like any kind of prefixes, but I can see why they are needed in the classes which are part of the public API - to prevent name clashes when subclassing things. The `GLRenderer` though is an internal class, so I didn't use any prefixing. I don't have a strong opinion on this though, so I can change it if you think it's more consistent.

Reply:
I would actually like to see some strategy in a project like OpenFL how things are formatted and named, but I also see your point.

We could come up with proper convention out of the things we now from other places and abstract something that works well for us and for OpenFL. Some best-practice for these helper variables. Ideas are of course welcome `;)`